Gemfile

```diff
@@ -56,3 +58,9 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 group :production do
   gem 'pg', '0.18.4'
 end
+
+# replaces npm uglifier which does not work on heroku
+gem 'uglifier'
+
+# added for integration test
+gem 'rails-controller-testing'
```

Comment:
Could you place these above the comment for `ENVIRONMENT-SPECIFIC GEMS` since they aren't environment-specific? E.g. `group :development...`.